Documentation/api-reference.md

```diff
@@ -129,7 +129,7 @@ sendEvent: function(xdmData as object, callback = _adb_default_callback as funct
 > **Note**
 > `sendEvent` API will automatically collect and attach the identities synced with the SDK and the implementation details. That data is sent as `IdentityMap` and `Implementation Details` fieldgroups under the XDM payload and are sent with every Experience Edge Event. IdentityMap is added to the schema automatically but if you would like to include ImplementationDetails information in your dataset, add the `Implementation Details` field group to the schema tied to your dataset.
```

Comment:
this note requires some adjustments and can we move it before the syntax:
The `sendEvent` API automatically attaches the following information with each Experience Event: 
- `IdentityMap` with the ECID - included by default to Experience Event class based XDM schemas
- `Implementation Details` - for more details see the [Implementation Details XDM field group](https://github.com/adobe/xdm/blob/master/components/datatypes/industry-verticals/implementationdetails.schema.json)